.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,47 @@
+name: Feature request
+description:  |
+  💡 Suggest an idea for the `NextUI` project
+  Examples
+    - propose a new component
+    - improve an exiting component
+    - ....etc
+body:
+  - type: markdown
+    attributes:
+      value: |
+        This issue form is for requesting features only! For example, requesting a new component, behavior ... etc
+        If you want to report a bug, please use the [bug report](https://github.com/nextui-org/nextui/issues/new?assignees=&labels=&template=bug_report.yml) form.
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
+      description: A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Describe the solution you'd like
+      description: A clear and concise description of what you want to happen.
+      placeholder: |
+        As a user, I expected ___ behavior but ___ ...
+        
+        Ideal Steps I would like to see:
+        1. Go to '...'
+        2. Click on '....'
+        3. ....
+  - type: textarea
+    validations:
+      required: true
+    attributes:
+      label: Describe alternatives you've considered
+      description: A clear and concise description of any alternative solutions or features you've considered.
+  - type: textarea
+    attributes:
+      label: Screenshots or Videos
+      description: |
+        If applicable, add screenshots or a video to help explain your problem.
+        For more information on the supported file image/file types and the file size limits, please refer
+        to the following link: https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/attaching-files
+      placeholder: |
+        You can drag your video or image files inside of this editor ↓
```

Comment:
Add Code of Conduct Checkbox here too

![image](https://user-images.githubusercontent.com/47073516/156115817-e64c7aa7-6b07-4e82-85d7-31beb0dffdb1.png)

OUTPUT:- 


![image](https://user-images.githubusercontent.com/47073516/156115876-bb238804-fb43-4eab-b3f4-92d0bf425d7c.png)